.github/workflows/ci.yml

```diff
@@ -294,20 +398,54 @@ jobs:
         run: |
           minikube profile list
           minikube status
-      - name: Build
+
+      # Download artifacts
+      - name: Download artifact - fluvio
+        uses: actions/download-artifact@v2
+        with:
+          name: fluvio-x86_64-unknown-linux-musl
+          path: .
+      - name: Download artifact - flv-test
+        uses: actions/download-artifact@v2
+        with:
+          name: flv-test-x86_64-unknown-linux-musl
+          path: .
+      - name: Download Docker Image as Artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: infinyon-fluvio
+          path: /tmp
+      - name: Load Fluvio Docker Image
         run: |
-          make RELEASE=release TARGET=x86_64-unknown-linux-musl build_test
+          ls -la /tmp
+          eval $(minikube -p minikube docker-env)
+          docker image load --input /tmp/infinyon-fluvio.tar
+          docker image ls -a
+
+      - name: Print artifacts and mark executable
+        run: ls -la . && chmod +x ./fluvio ./flv-test && ./fluvio -h && ./flv-test -h
+
       - name: Setup installation pre-requisites
         run: |
-          make RELEASE=true TARGET=x86_64-unknown-linux-musl  k8-setup
-      - name: Make image
-        run: make RELEASE=true minikube_image
+          make FLUVIO_BIN=./fluvio RELEASE=true TARGET=x86_64-unknown-linux-musl k8-setup
+
       - name: Run smoke-test-k8-tls-root
         timeout-minutes: 5
         run: |
-          make RELEASE=true TARGET=x86_64-unknown-linux-musl UNINSTALL=noclean smoke-test-k8-tls-root
+          kubectl create configmap authorization --from-file=POLICY=./src/sc/test-data/auth_config/policy.json --from-file=SCOPES=./src/sc/test-data/auth_config/scopes.json
```

Comment:
don't duplicate makefile steps otherwise this will cause issue when makefile is changed

Reply:
I was going to ask about this. The Make recipe that was run here previously was `make smoke-test-k8-tls-root`, but in the dependencies of that Make task, it also executes `build_test` which executes `build_cluster` and `build_cli`. So what ends up happening is a full rebuild of all our components, when what we actually want is to just run the version of `flv-test` that we just downloaded from the artifact cache.

I will see if there is a way to prevent running the dependencies for the Make recipe, but this was done specifically to prevent a full rebuild.